src/components/slider/slider.scss

```diff
@@ -117,7 +117,6 @@ $colorGrey: #999999;
 
 .mdc-slider__thumb-container {
     top: 0;
-    z-index: 1;
```

Comment:
@Kiarokh This z-index was introduced by you in #1018. Can you test this and make sure something else doesn't break from removing it?

(If we can remove it, that's really good. The fewer instances of z-index we need, the better.)

Reply:
> @Kiarokh This z-index was introduced by you in #1018. Can you test this and make sure something else doesn't break from removing it?
> 
> (If we can remove it, that's really good. The fewer instances of z-index we need, the better.)

I let @Kiarokh know about it and he was fine with it. Can you confirm it here?

Reply:
![slider](https://user-images.githubusercontent.com/35954987/103904368-78e2a780-50fd-11eb-9904-8e2c21766d41.gif)
the z-index was there to make sure that the `mdc-slider__thumb-container` isn't covering the visualizations of start and end point. :(

Reply:
I actually don’t think we need to fix this bug, since it seems to only happen in that specific Lime Elements example. Something we understood after investigations. 

Portal adds a dynamic z-index on dropdown menus. Which seems to be always higher than 1. So in the webclient this issue won't happen.